client/lib/src/server.rs

```diff
@@ -1,80 +1,70 @@
+#![allow(missing_docs)]
```

Comment:
I prefer to allow this on individual things that are clearly self-documenting, rather than entire modules.